.travis.yml

```diff
@@ -1,22 +1,20 @@
-sudo: false
-dist: trusty
+dist: xenial
 language: node_js
-cache:
-  directories:
-    - node_modules
+# node version is specified using the .nvmrc file
+cache: npm
 before_install:
   - npm install -g greenkeeper-lockfile@1
 before_script:
   # pulseaudio is needed for dockerized chrome since it has a dummy sink and
   # the dummy audio ALSA kernel module isn't loaded by Travis
   # libavcodec54 is needed by firefox for h.264 support
-  - sudo apt-get install --no-install-recommends pulseaudio libavcodec54
+  - sudo apt-get install pulseaudio
```

Comment:
don't we need libavcodec54 for firefox?

Reply:
not in xenial it seems, but regardless I am backing out all the test changes. I just can't get things to work and they don't really belong in this branch.